dataRetriever/resolvers/topicResolverSender/diffPeerListCreator_test.go

```diff
@@ -138,7 +138,7 @@ func TestDiffPeerListCreator_PeersListEmptyMainListShouldRetEmpty(t *testing.T)
 		excludedTopic,
 	)
 
-	assert.Empty(t, dplc.PeerList())
+	assert.Empty(t, dplc.CrossShardPeerList())
```

Comment:
Please rename also the test names. Valid for L127, L144, L164, L196. Please remove the TODO on L125

Reply:
done